src/components/Dashboard.tsx

```diff
@@ -48,26 +48,37 @@ const Dashboard: React.FC = () => {
 }
 
 const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template }) => {
-  const { name, code, hasApplyPermission, filters } = template
+  const { plural, code, hasApplyPermission, filters, permissions } = template
+
+  const userRole =
+    permissions.filter((type) => type === PermissionPolicyType.Apply).length > 0
+      ? USER_ROLES.APPLICANT
+      : USER_ROLES.REVIEWER
+
   return (
     <div className="template">
-      <div className="title">
-        <Header as="h5">{name}</Header>
-
+      <div className="content">
+        <div className="filters">
+          <Label
+            className="strong-label"
+            as={Link}
+            to={`/applications?type=${code}&user-role=${userRole}`}
+          >
+            {plural}
```

Comment:
I really don't like the "Plural" thing (see my back-end comment), but if we are going to have it, can it at least have a fallback to the "name" if not present?

But again, I really think we should let the localisation library handle this, not manually specify every plural.